HappyCamps/src/app/Models/roles.ts

```diff
@@ -0,0 +1,5 @@
+export enum Roles{
+    VOLUNTEER = 0,
+    ORGANIZER = 1,
+    ADMIN = 2,
```

Comment:
Pretty sure that it's pointless to initialize the enum with values

Reply:
I'm going to remove